src/client/keycode.cpp

```diff
@@ -6,15 +6,17 @@
 #include "settings.h"
 #include "log.h"
 #include "debug.h"
+#include "renderingengine.h"
 #include "util/hex.h"
 #include "util/string.h"
 #include "util/basic_macros.h"
+#include <vector>
 
 struct table_key {
-	const char *Name;
+	std::string Name;
```

Comment:
While we're already at it, a small improvement?
```suggestion
	std::string Name; // An EKEY_CODE 'symbol' name as a string
```